tests/components/apprise/test_notify.py

```diff
@@ -0,0 +1,110 @@
+"""The tests for the apprise notification platform."""
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+from homeassistant.setup import async_setup_component
+
+BASE_COMPONENT = "notify"
+
+
+async def test_apprise_config_load_fail01(hass):
+    """Test apprise configuration failures 1."""
+
+    config = {
+        BASE_COMPONENT: {"name": "test", "platform": "apprise", "config": "/path/"}
+    }
+
+    with patch("apprise.AppriseConfig.add", return_value=False):
+        assert await async_setup_component(hass, BASE_COMPONENT, config)
+        await hass.async_block_till_done()
+
+        # Test that our service failed to load
+        assert hass.services.has_service(BASE_COMPONENT, "test") is False
+
+
+async def test_apprise_config_load_fail02(hass):
+    """Test apprise configuration failures 2."""
+
+    config = {
+        BASE_COMPONENT: {"name": "test", "platform": "apprise", "config": "/path/"}
+    }
+
+    with patch("apprise.Apprise.add", return_value=False):
+        with patch("apprise.AppriseConfig.add", return_value=True):
+            assert await async_setup_component(hass, BASE_COMPONENT, config)
+            await hass.async_block_till_done()
+
+            # Test that our service failed to load
+            assert hass.services.has_service(BASE_COMPONENT, "test") is False
```

Comment:
```suggestion
            assert not hass.services.has_service(BASE_COMPONENT, "test")
```

Reply:
I applied your suggestion to all references :slightly_smiling_face: .  This one is also resolved as the last identified below.